app/schemas/FOOD.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "foodId": {
+      "type": "string"
+    },
+    "name": {
+      "type": "string"
+    },
+    "description": {
+      "type": "string"
+    },
+    "nutritionalData": {
```

Comment:
Can you please make a Markdown Table for `NutrionalData`? I think we're going to be using this a lot and we'll need a quick reference.

Reply:
Sure